app/components/pool/action/scale/autoscale-formula-picker.component.ts

```diff
@@ -47,6 +51,8 @@ export class AutoscaleFormulaPickerComponent implements OnInit, OnDestroy, Contr
         this._subs.push(this.autoscaleFormulaService.formulas.subscribe((formulas) => {
             this.savedAutoscaleFormulas = formulas;
         }));
+        this.predefinedFormulaService.init();
```

Comment:
I would move this to the `app.component` That's static enough so you don't want to reload every time